models/activity.ts

```diff
@@ -0,0 +1,55 @@
+import { list } from "@keystone-6/core";
+import type { ListConfig } from "@keystone-6/core";
+import type { Lists } from ".keystone/types";
+import {
+  integer,
+  text,
+  timestamp,
+  relationship,
+} from "@keystone-6/core/fields";
+
+export const Activity: ListConfig<Lists.Activity.TypeInfo<any>, any> = list({
+  access: {
+    operation: {
+      query: ({ session }) => !!session,
+      create: ({ session }) => !!session,
+      update: ({ session }) => !!session,
+      delete: ({ session }) => !!session,
+    },
+    item: {
+      update: ({ session, item }) => item.userId === session.data.id,
+      delete: ({ session, item }) => item.userId === session.data.id,
+    },
+  },
+  fields: {
+    name: text({
+      defaultValue: "...",
+      db: { map: "my_text", nativeType: "VarChar(40)" },
+      validation: { isRequired: true },
+      isIndexed: "unique",
+      ui: { displayMode: "textarea" },
+    }),
```

Comment:
I am actually not sure if you need all these db: references on something simple like a text or integer field. This isn't how I've seen models setup in keystone/apollo but that doesn't necessarily mean it's wrong. Just a thought on this.

Reply:
I looked at other PRs and made mine match more so to them. I did look up the keystone fields and the examples they gave all had those references which is why I used them but the code looks cleaner now.